crates/core/src/operations/mod.rs

```diff
@@ -199,6 +201,12 @@ impl DeltaOps {
     pub fn add_constraint(self) -> ConstraintBuilder {
         ConstraintBuilder::new(self.0.log_store, self.0.state.unwrap())
     }
+
+    /// Drops constraints from a table
+    #[must_use]
```

Comment:
```suggestion
   #[cfg(feature = "datafusion")]
    #[must_use]
```

This should be guarded on the `datafusion` feature

Reply:
Ah oops :)